lib/charms/mongodb/v1/mongos.py

```diff
@@ -334,6 +334,36 @@ def _log_removal_info(self, removal_info, shard_name):
             ",".join(dbs_to_move),
         )
 
+    @property
+    def is_ready(self) -> bool:
+        """Is mongos ready for services requests.
+
+        Returns:
+            True if services is ready False otherwise. Retries over a period of 60 seconds times to
+            allow server time to start up.
+
+        Raises:
+            ConfigurationError, ConfigurationError, OperationFailure
+        """
+        try:
+            for attempt in Retrying(stop=stop_after_delay(60), wait=wait_fixed(3)):
+                with attempt:
+                    # The ping command is cheap and does not require auth.
+                    self.client.admin.command("ping")
```

Comment:
Des this call when ping failing?

Reply:
When ping fails an exception is raised :)